bigbluebutton-html5/private/config/settings.yml

```diff
@@ -307,6 +307,9 @@ public:
   # plugins:
   #   - name: SamplePresentationToolbarPlugin
   #     url: https://<your-Host>/plugins/SamplePresentationToolbarPlugin.js
+    # Timeout before automatically redirecting users to the logout URL (or closing the window if the session is
```

Comment:
Please move this new comment+config to be a few lines higher (above plugins:) to avoid yml structure errors